okhttp/src/jvmTest/java/okhttp3/internal/publicsuffix/PublicSuffixDatabaseTest.kt

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (C) 2017 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal.publicsuffix
+
+import okhttp3.internal.toCanonicalHost
+import okio.Buffer
+import okio.GzipSource
+import okio.buffer
+import okio.source
+import okio.use
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class PublicSuffixDatabaseTest {
```

Comment:
Migrating the database itself is annoying due to its use of Java resource loading! Finally time to address that in Okio??